Exporters/3ds Max/BabylonExport.Entities/BabylonExport.Entities.csproj

```diff
@@ -78,19 +81,20 @@
     <Compile Include="BabylonSound.cs" />
     <Compile Include="BabylonSubMesh.cs" />
     <Compile Include="BabylonTexture.cs" />
+    <Compile Include="BabylonUniversalCamera.cs" />
     <Compile Include="BabylonVector3.cs" />
     <Compile Include="Properties\AssemblyInfo.cs" />
   </ItemGroup>
   <ItemGroup>
     <None Include="packages.config" />
   </ItemGroup>
   <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
-  <Import Project="..\packages\SharpDX.2.6.3\build\SharpDX.targets" Condition="Exists('..\packages\SharpDX.2.6.3\build\SharpDX.targets')" />
+  <Import Project=".\packages\SharpDX.2.6.3\build\SharpDX.targets" Condition="Exists('.\packages\SharpDX.2.6.3\build\SharpDX.targets')" />

```

Comment:
There is NO solution file for the project... If you open just the csproj when you refresh Nuget, it will create the 'packages' folder in the project folder (Same folder as csproj). The original reference MUST have had a solution file a the parent of the folder '..\'.

But now, there is no solution everything runs out the main project folder